voice/bxml/verbs/speakSentence.md

```diff
@@ -68,7 +69,7 @@ Attributes:
  * `level`: (optional) defines the strength of emphasis to be applied, accepted values are: `strong`, `moderate` or `reduced`
 
 #### `<lang>`
-Specifies the natural language of the content.
+Specifies the natural language of the content.  For example, if you use an English American voice to speak a Spanish sentence, it will say the Spanish words with an American accent. 
```

Comment:
```suggestion
Specifies the natural language of the content. For example, a voice with an `en_us` locale will pronounce an `es-US` sentence like an American who can't speak Spanish.
```
I think it's important to note that this is more than just an accent (e.g., the difference between `es_mx` and `es_us`).